extensions/omni_httpd/tests/create_drop.yml

```diff
@@ -10,5 +10,40 @@ tests:
   tests:
   - query: create extension omni_httpd cascade
     commit: true
+
+    # https://github.com/omnigres/omnigres/issues/479
+  - query: select count(*)
+           from pg_ls_dir((select setting from pg_settings where name = 'data_directory'))
+           where pg_ls_dir like 'omni_httpd%'
+    results:
+      - count: 0
   - query: drop extension omni_httpd
     commit: true
+
+- name: temp_dir doesn't exist
+  transaction: false
+  query: alter system set omni_httpd.temp_dir = '/does-not-exist'
+  error:
+    severity: ERROR
+    message: "'/does-not-exist' temp directory does not exist."
+
+- name: temp_dir inside data dir
+  transaction: false
+  query: |
+    do
+    $$
+        declare
+            data_dir text;
+        begin
+            select setting
+            into data_dir
+            from pg_settings
+            where name = 'data_directory';
+            
+            execute format('alter system set omni_httpd.temp_dir = ''%s''', data_dir);
```

Comment:
this test fails with `ALTER SYSTEM cannot be executed from a function`
@yrashk any ideas on how to execute 'ALTER SYSTEM` with dynamic value ?

Reply:
may be call `set_config(...)` instead?

Reply:
doesn't work
`ERROR:  parameter "omni_httpd.temp_dir" cannot be changed now`

Reply:
it has been tested manually, marked it as todo